src/main/java/codeu/view/ComponentProvider.java

```diff
@@ -0,0 +1,205 @@
+package codeu.view;
```

Comment:
So this file is to replace activity-helper.jsp? I like that it makes formatting so much neater!